runtime/gc_glue_java/MetronomeDelegate.hpp

```diff
@@ -157,10 +166,374 @@ class MM_MetronomeDelegate : public MM_BaseNonVirtual
 	void acquireExclusiveVMAccess(MM_EnvironmentBase *env, bool waitRequired);
 	void releaseExclusiveVMAccess(MM_EnvironmentBase *env, bool releaseRequired);
 
+	void markLiveObjects(MM_EnvironmentRealtime *env);
+#if defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING)
+	MMINLINE void markClassOfObject(MM_EnvironmentRealtime *env, J9Object *objectPtr)
```

Comment:
Style - new look: for functions declared in .hpp files we should use the same style as in .cpp - have declarated types in separate line (at least for new code):
```
MMINLINE void 
markClassOfObject(MM_EnvironmentRealtime *env, J9Object *objectPtr)
```
If it is convenient for you, would you please use this format for this file?